include/GLFW/glfw3.h

```diff
@@ -649,6 +649,7 @@ extern "C" {
 #define GLFW_SRGB_CAPABLE           0x0002100E
 #define GLFW_REFRESH_RATE           0x0002100F
 #define GLFW_DOUBLEBUFFER           0x00021010
+#define GLFW_ALPHA_MASK             0x00021011
```

Comment:
@elmindreda Just fixed.  There are two commits, one for `GLFW_ALPHA_MASK` (69fca03d781c125e38d5b6c59448d0496231c3b6), and one for  `alphaMask` (c8c7ad249a682590e80eb383b07771692ce8dbfa)